tests/helper/test_helper.py

```diff
@@ -153,32 +153,43 @@ def test_index_starred(mocker,
     assert index_messages(messages, model, model.index) == expected_index
 
 
-@pytest.mark.parametrize('mentioned_messages', [
-    {537286, 537287, 537288},
-    {537286}, {537287}, {537288},
-    {537286, 537287}, {537286, 537288}, {537287, 537288},
+@pytest.mark.parametrize(['mentioned_messages',
+                          'wildcard_mentioned_messages'], [
```

Comment:
With extra line-length available and the way `black` reformats these lines (indented on the follow line), we should generally have plenty of space available and not need a list for the params compared to just a single string.